### PR TITLE
Worst offense

### PR DESCRIPTION
**Pull request by ParkerLockhart**
Those helpers from the last method came in really handy.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -31,25 +31,46 @@ def self.from_csv(locations)
...
+    end
+    goals.sum
+  end
 
+  def average_goals_per_game(team_id)
+    total_goals_by_team(team_id).to_f / games_by_team(team_id).length.to_f
+  end
 
+  def worst_offense
+    id = id = @game_teams.min_by do |team|
```
Comment (jbreit88) [inline lib/stat_tracker.rb]: hmmmm....you assign the ID variable twice here.  I don't think it hurts your code or changes anything, but we should probably clean that up anyways. :)

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -24,14 +24,53 @@
     it 'exists' do
       expect(@stat_tracker).to be_instance_of(StatTracker)
     end
-end
-# Game Statistics Tests
 
+    describe '#count of teams' do
```
Comment (jbreit88) [inline spec/stat_tracker_spec.rb:29]: Is this indentation a little off? I can't tell for sure on GH

**Review by jbreit88 — CHANGES_REQUESTED** [review]
Overall this all looks good. there were just two very small places I saw that we might clean up really quick.

File: lib/stat_tracker.rb (hunk 1 above)
Comment (ParkerLockhart) [reply lib/stat_tracker.rb]: Oooop, good catch! That's what I get for trying to write code after midnight haha. Eagle eyes at it again, thank you!!

File: spec/stat_tracker_spec.rb (hunk 2 above)
Comment (ParkerLockhart) [reply spec/stat_tracker_spec.rb:29]: Took a look at it in my IDE and I think my 'exists' test block was in one indent too far, but I also didn't put a separate describe block for it and I think that would make it look more uniform and easier to read with the rest of the code. I'll go head and throw that in there and let me know if it looks better!

**Review by joel-grant — APPROVED** [review]
I'm loving the multiple tests!

File: spec/stat_tracker_spec.rb (hunk 2 above)
Comment (jbreit88) [reply spec/stat_tracker_spec.rb:29]: Thank you. Yeah, it's probably on my end, not yours.

**Review by jbreit88 — APPROVED** [review]

**Comment by jgrazulis** [comment]
Nice edits all!

**Pull request merged**